daprdocs/layouts/shortcodes/dapr-latest-version.html

```diff
@@ -0,0 +1 @@
+1.4.0
```

Comment:
Since some places expect 1.4 and some 1.4.0 I think this should be 1.4 and the where needed add the suffix .0

Reply:
I actually created a conditional in the short-code to handle this based on params being set. It's cleaner than leaving .0's around the markdown.